snappy/snapp_snapfs_test.go

```diff
@@ -292,7 +250,7 @@ func (s *SquashfsTestSuite) TestInstallKernelSnapUnpacksKernel(c *C) {
 		OfficialName: "ubuntu-kernel",
 		Revision:     snap.R(42),
 	}
-	_, err := (&Overlord{}).InstallWithSideInfo(snapPkg, si, 0, &MockProgressMeter{})
+	_, err := (&Overlord{}).InstallWithSideInfo(snapPkg, si, LegacyInhibitHooks, &MockProgressMeter{})
```

Comment:
why these changes? Was the old test broken in that it didn't have inhibit hooks set?